boards/pic32-wifire/include/periph_conf.h

```diff
@@ -33,15 +33,15 @@ extern "C" {
 #define PERIPHERAL_CLOCK (100000000)  /* Hz */
 
 /**
- * @brief   Timer definitions
+ * @name   Timer definitions
  * @{
  */
 #define TIMER_NUMOF         (1)
 #define TIMER_0_CHANNELS    (3)
 /** @} */
 
 /**
-  * @brief   UART Definitions
+  * @name   UART Definitions
```

Comment:
At least in cases like this proper alignment would be appreciated. This just looks ugly ;-).